src/es-module-shims.js

```diff
@@ -163,8 +166,19 @@ function resolveDeps (load, seen) {
     resolvedSource += source.slice(lastIndex);
   }
 
-  if (resolvedSource.indexOf('//# sourceURL=') === -1)
+  let hasSourceMappingURL = false
+  resolvedSource = resolvedSource.replace(/\/\/# sourceMappingURL=(.*)\s*$/, (match, url) => {
+    hasSourceMappingURL = true
+    return match.replace(url, new URL(url, load.r))
+  });
+  let hasSourceURL = false
+  resolvedSource = resolvedSource.replace(/\/\/# sourceURL=(.*)\s*$/, (match, url) => {
+    hasSourceURL = true
+    return match.replace(url, new URL(url, load.r))
+  });
+  if (!hasSourceMappingURL && !hasSourceURL) {
```

Comment:
Is there an actual issue with adding `sourceURL` if there is a `sourceMappingURL`? I recall using them together in workflows in the past just fine. Might be useful eg when source maps are disabled - would be worth checking? Or did this definitely stop something from working?

Reply:
Hm, I didn't break anything for me AFAIK, let me test the behavior of different browsers in this case, like which one takes precedence in the dev tools etc.

Reply:
Just don't forget to check this if you can... I think we might well be able to remove the `hasSourceURL` check.

Reply:
yeah it works fine, at least in Chrome and Safari, so I removed that part of the condition